.vortex/tooling/src/helpers.php

```
@@ -549,6 +644,147 @@ function lagoon_exec(string $bin, string $subcommand, array $ctx, ?int &$exit_co
+
+  if (!is_dir($home)) {
+    mkdir($home, 0755, TRUE);
+    // The isolated home caches acli's token and state; remove it when the run
+    // ends so no credentials linger on disk afterwards.
+    register_shutdown_function(static function () use ($home): void {
+      // @codeCoverageIgnoreStart
+      if (!is_dir($home)) {
+        return;
+      }
+
+      $items = new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($home, \RecursiveDirectoryIterator::SKIP_DOTS), \RecursiveIteratorIterator::CHILD_FIRST);
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '717', column '20'). (undefined)

(MissingImport)